tests/src/Common/Platform/platformdefines.h

```diff
@@ -55,6 +55,11 @@ typedef const WCHAR *LPCWSTR, *PCWSTR;
 #define __stdcall
 #define _cdecl
 #endif
+#ifndef CROSS_MSC_VER_DEFINES
+#define CROSS_MSC_VER_DEFINES
+#define __FUNCSIG__ "<clang-3.5 signature not supported>"
+#define __int64 long long
+#endif // CROSS_MSC_VER_DEFINES
```

Comment:
clang 3.5 as well. If you just run ./build.sh in the coreclr repo, it builds all the test binaries together with the coreclr native components:
```
ls -la ~/git/coreclr/bin/Product/Linux.x64.Debug/bin/
drwxrwxr-x 2 janvorli janvorli   4096 bře 21 16:49 .
drwxrwxr-x 9 janvorli janvorli   4096 bře 21 16:50 ..
-rw-r--r-- 1 janvorli janvorli 144180 bře 21 16:44 hfa_nested_f32_native_cpp.so
-rw-r--r-- 1 janvorli janvorli 152420 bře 21 16:44 hfa_nested_f64_native_cpp.so
-rw-r--r-- 1 janvorli janvorli 107500 bře 21 16:44 hfa_simple_f32_native_cpp.so
-rw-r--r-- 1 janvorli janvorli 107436 bře 21 16:44 hfa_simple_f64_native_cpp.so
-rw-r--r-- 1 janvorli janvorli  31600 bře 21 16:44 jitstructtests_lib.so
-rw-r--r-- 1 janvorli janvorli  31017 bře 21 16:44 libBestFitMappingNative.so
-rw-r--r-- 1 janvorli janvorli  17517 bře 21 16:44 libBoolNative.so
-rw-r--r-- 1 janvorli janvorli 137428 bře 21 16:44 libForeignThreadExceptionsNative.so
-rw-r--r-- 1 janvorli janvorli  25934 bře 21 16:44 libFuncPtrAsDelegateParamNative.so
-rw-r--r-- 1 janvorli janvorli  16423 bře 21 16:44 libFunctionPointerNative.so
-rw-r--r-- 1 janvorli janvorli  16172 bře 21 16:44 libIUnknownNative.so
-rw-r--r-- 1 janvorli janvorli  23904 bře 21 16:44 libLPSTRTestNative.so
-rw-r--r-- 1 janvorli janvorli  22869 bře 21 16:44 libLPTSTRTestNative.so
-rw-r--r-- 1 janvorli janvorli  68532 bře 21 16:44 libMarshalArrayByValNative.so
-rw-r--r-- 1 janvorli janvorli  22193 bře 21 16:44 libMarshalBoolArrayNative.so
-rw-r--r-- 1 janvorli janvorli  16429 bře 21 16:44 libMarshalEnumNative.so
-rw-r--r-- 1 janvorli janvorli 110371 bře 21 16:44 libMarshalStructAsParam.so
-rw-r--r-- 1 janvorli janvorli  16176 bře 21 16:44 libNativeCallableDll.so
-rw-r--r-- 1 janvorli janvorli  17688 bře 21 16:44 libRefCharArrayNative.so
-rw-r--r-- 1 janvorli janvorli  17448 bře 21 16:44 libRefIntNative.so
-rw-r--r-- 1 janvorli janvorli  17327 bře 21 16:44 libSimpleStructNative.so
-rw-r--r-- 1 janvorli janvorli  16240 bře 21 16:44 libSizeConstNative.so
-rw-r--r-- 1 janvorli janvorli  26168 bře 21 16:44 libStructABILib.so
-rw-r--r-- 1 janvorli janvorli  17330 bře 21 16:44 libUIntPtrNative.so
-rw-r--r-- 1 janvorli janvorli  23607 bře 21 16:44 libUTF8TestNative.so
-rw-r--r-- 1 janvorli janvorli  24250 bře 21 16:44 libVector3TestNative.so
-rw-r--r-- 1 janvorli janvorli   9747 bře 21 16:44 mirror.so
-rw-r--r-- 1 janvorli janvorli  10293 bře 21 16:44 native_i0c.so
-rw-r--r-- 1 janvorli janvorli   9753 bře 21 16:44 native_i0s.so
-rw-r--r-- 1 janvorli janvorli  10485 bře 21 16:44 native_i1c.so
-rw-r--r-- 1 janvorli janvorli   9953 bře 21 16:44 native_i1s.so
-rw-r--r-- 1 janvorli janvorli  11098 bře 21 16:44 native_i3c.so
-rw-r--r-- 1 janvorli janvorli  15406 bře 21 16:44 native_i3s.so
-rw-r--r-- 1 janvorli janvorli  15429 bře 21 16:44 native_i5c.so
-rw-r--r-- 1 janvorli janvorli  15649 bře 21 16:44 native_i5s.so
-rw-r--r-- 1 janvorli janvorli  15669 bře 21 16:44 native_i6c.so
-rw-r--r-- 1 janvorli janvorli  15881 bře 21 16:44 native_i6s.so
-rw-r--r-- 1 janvorli janvorli  29125 bře 21 16:44 native_i7c.so
-rw-r--r-- 1 janvorli janvorli  29329 bře 21 16:44 native_i7s.so
-rw-r--r-- 1 janvorli janvorli  10802 bře 21 16:44 native_i8c.so
-rw-r--r-- 1 janvorli janvorli  15054 bře 21 16:44 native_i8s.so
-rw-r--r-- 1 janvorli janvorli   9783 bře 21 16:44 test2.so
```